app/src/pages/inside/launchesPage/modals/launchMergeModal/launchMergeModal.jsx

```diff
@@ -214,11 +217,15 @@ export class LaunchMergeModal extends Component {
     fetch(URLS.launchesMerge(this.props.activeProject), {
       method: 'post',
       data: values,
-    }).then(() => {
-      this.props.data.fetchFunc();
-      closeModal();
-      this.props.hideScreenLockAction();
-    });
+    })
+      .then(() => {
+        this.props.data.fetchFunc();
+        closeModal();
+      })
+      .catch(this.props.showDefaultErrorNotification)
+      .finally(() => {
```

Comment:
Doesn't work in IE: [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally)

Reply:
Fixed